keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -235,10 +234,10 @@ legacy tokens for security practices.
 
 #### Alpha -> Beta Graduation
 
-- [ ] In use by multiple distributions
-- [ ] Approved by PRR and scalability
-- [ ] Any known bugs fixed
-- [ ] Tests passing
+- [x] In use by multiple distributions
```

Comment:
I don't think we can claim this one yet, and I'd actually have expected this to be a beta → GA graduation criteria (it's listed there... I'd recommend dropping this line from the alpha → beta list)

Reply:
that's right. done.